### DISCUSSION TITLE
Run howler in pure NodeJS environment ?

### DISCUSSION THREAD

**Discussion by chaperot-perso** [goldfire/howler.js#1608] [Q&A]
Hi to everyone 😊
I would like to use howler in a **pure nodejs** environment, to play sounds on Raspberry's sound card, or external USB card...
Is it possible ?
Thanks for your help 🙏 💖

**Comment by PhP60** [comment]
Hi !!
I got the same question. I found some documentation on that, but nothing sounds good !!
Thanks....
